frontend/src/services/Api.ts

```diff
@@ -706,6 +706,7 @@ export const getClustersAppHealth = async (
             if (namespaceAppHealth[ns][k]) {
               const ah = AppHealth.fromJson(namespaces, k, namespaceAppHealth[ns][k], {
                 rateInterval: duration,
+                // @TODO replace hardcoded values
```

Comment:
If this is hardcoded and is not available, and now it check if health is not empty, can this be this optional and remove it from here?

Reply:
good idea, removed